xml/security_ldap.xml

```diff
@@ -861,6 +788,99 @@ systemctl start sssd</screen>
  </sect2>
 </sect1>
 
+<!-- default installation uses self-signed cert -->
+ <sect1 xml:id="sec-security-ldap-server-ca">
+ <title>Using CA Certificates for TSL</title>
+ <para>
+  You can manage the CA certificates for &ds389; with the following command
+  line tools: <command>certutil</command>, <command>openssl</command>, and
+  <command>pk12util</command>.
+ </para>
+ <para>
+  For testing purposes, you can create a self-signed certificate with
```

Comment:
I think here maybe you mean the server automatically generates a self signed cert during instance creation, rather than "you can create"?

Reply:
I haven't gotten to the TLS section yet, this is the old version.